mathesar_ui/src/components/index.ts

```diff
@@ -26,3 +26,4 @@ export { default as Pagination } from './pagination/Pagination.svelte';
 export { default as Dropdown } from './dropdown/Dropdown.svelte';
 export { default as Select } from './select/Select.svelte';
 export { default as FileUpload } from './file-upload/FileUpload.svelte';
+export { default as SchemaRow } from './schema-row/SchemaRow.svelte';
```

Comment:
This is a component specific to schemas page. So, it would be better to place it within `pages/schemas`.

The `components` folder is only for re-usable components. The plan is to spin if off as a separate package, as a fully fledged component library, after the MVP.

Reply:
Understood 👍. Fixed in 775f800.